src/core/Slider/Thumb.tsx

```diff
@@ -0,0 +1,101 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { Tooltip, TooltipProps } from '../Tooltip';
+import { CommonProps } from '../utils/props';
+
+/**
+ * Thumb is a local component used to show and modify the values maintained by the Slider.
+ * Only one Thumb can be active at a time. A Thumb is made active when the user selects
+ * it with pointer. Whenever a Thumb is active, focused, or hovered its tooltip is shown.
+ */
+export const Thumb = ({
+  value,
+  index,
+  minVal,
+  maxVal,
+  step,
+  sliderMin,
+  sliderMax,
+  isActive,
+  onThumbActivated,
+  onThumbValueChanged,
+  tooltipContent,
+  tooltipProps,
+  thumbProps,
+}: {
+  value: number;
+  index: number;
+  minVal: number;
+  maxVal: number;
+  step: number;
+  sliderMin: number;
+  sliderMax: number;
+  isActive: boolean;
+  onThumbActivated: (index: number) => void;
+  onThumbValueChanged: (index: number, value: number) => void;
+  tooltipContent: React.ReactNode;
+  tooltipProps?: Partial<Omit<TooltipProps, 'content' | 'children'>>;
+  thumbProps?: Omit<CommonProps, 'title'>;
```

Comment:
I wonder if this should be `(index) => React.HTMLAttributes<HTMLDivElement>`.

It would give two things:
- granular control over multiple thumbs
- any arbitrary props can be passed (such as custom data/aria attributes or events).